CHANGELOG

```diff
@@ -46,6 +46,7 @@ Miscellaneous
   This parameter can still be explicitly overridden with
   VisItDataCollection::SetPadDigitsRank().
 
+- Added a constructor for QuadratureFunction with external data.
```

Comment:
Probably not needed in the CHANGELOG…

Reply:
Thanks -- I wasn't sure -- I can remove it.

Reply:
I removed it.